src/core/utils/chrom-size.ts

```diff
@@ -187,6 +187,25 @@ export const CHROM_SIZE_MM9 = {
     chrM: 16299
 };
 
+export const CHROM_SIZE_SACCER3 = {
+    chrI: 230218,
+    chrII: 813184,
+    chrIII: 316620,
+    chrIV: 1531933,
+    chrV: 576874,
+    chrVI: 270161,
+    chrVII: 1090940,
+    chrVIII: 562643,
+    chrIX: 439888,
+    chrX: 745751,
+    chrXI: 666816,
+    chrXII: 1078177,
+    chrXIII: 924431,
+    chrXIV: 784333,
+    chrXV: 1091291,
+    chrXVI: 948066
+};
+
```

Comment:
The Yeast assembly is added to Gosling, just so that we do not need to define the chromosome names and sizes in example specs. Now, we can use `assembly: 'sacCer3'`.